addons/Makefile

```diff
@@ -11,7 +11,18 @@ ROOT_DIR	:= $(dir $(shell pwd))
 export GO111MODULE := on
 
 # Image URL to use all building/pushing image targets
-IMG ?= addons-controller:$(shell git describe --always --dirty --tags)
+#IMG ?= addons-controller:$(shell git describe --always --dirty --tags)
```

Comment:
nit: i think this commented line can be removed

Reply:
removed this line